### DISCUSSION TITLE
no license information in json_fwd.hpp

### DISCUSSION THREAD

**Discussion by berberic2** [nlohmann/json#2454] [Ideas]
There is no license information in **_json_fwd.hpp_**

I wanted to add the single-file version and the forward-header to an (open-source) project and have been asked to add license-information into **_json_fwd.hpp_**.

Wouldn’t it be useful to add a single-line license-information to this file, to prevent confusion?

**Comment by nlohmann** [comment]
The MIT license not really requires adding the license to all files, but I may work on this in the future.
